File: src/codes/Users_Manual/geometry_1.cpp
//~#include<chrono>
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
autd3::ControllerBuilder(
    {autd3::AUTD3(autd3::Point3(-autd3::AUTD3::DEVICE_WIDTH, 0, 0)),
     autd3::AUTD3(autd3::Point3::origin())})
    //~;
    //~return 0; }

File: src/codes/Users_Manual/geometry_1.py
~from pyautd3 import AUTD3, Controller
~from pyautd3.link.nop import Nop
~link = Nop()
Controller.open(
    [
        AUTD3(pos=[-AUTD3.DEVICE_WIDTH, 0.0, 0.0], rot=[1, 0, 0, 0]),
        AUTD3(pos=[0.0, 0.0, 0.0], rot=[1, 0, 0, 0]),
    ],
    link,
)